shared-files/_glossary.md

```diff
@@ -0,0 +1,324 @@
+    <dd>
+        The section of the Rancher UI where you manage Helm chart repositories. In earlier versions of Rancher, this section was called <i>Catalogs</i>. In later versions, it is called <i>Apps</i>.
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Apps, Catalogs</i>
+    </p>
+</dl>
+
+## C
+
+<dl>
+    <dt>
+        Catalogs
+    </dt>
+    <p>
+        <b>Versions:</b> 1. Rancher v2.4.x and earlier 2. Rancher v2.7.x and later
+    </p>
+    <dd>
+        <ol>
+            <li>
+                The section of the Rancher UI where you manage Helm chart repositories. In later versions of Rancher, this section is called <i>Apps & Marketplace</i>, or <i>Apps</i>.
+            </li>
+            <li>
+                Extension catalogs, container image repositories that can be imported to add extensions to your Rancher instance.
+            </li>
+        </ol>
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Apps, Apps & Marketplace</i>
+    </p>
+    <dt>
+        Cloud credentials
+    </dt>
+    <dd>
+        Placeholder
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Hosted clusters</i>
+    </p>
+    <dt>
+        Community
+    </dt>
+    <dd>
+        A build of Rancher that's available to the entire Rancher community.
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Prime, Rancher Prime</i>
+    </p>
+</dl>
+
+## D
+
+<dl>
+    <dt>
+        Downstream cluster
+    </dt>
+    <dd>
+        An alternative name for a <i>Managed</i> cluster: a Kubernetes cluster that is managed by a Rancher server.
+    </dd>
+    <p>
+        <b>Synonyms:</b> <i>Managed cluster</i>
+    </p>
+    <p>
+        <b>Related terms:</b> <i>Hosted cluster, Imported cluster, Rancher server, Registered cluster</i>
+    </p>
+</dl>
+
+## E
+
+<dl>
+    <dt>
+        Extension
+    </dt>
+    <p>
+        <b>Versions:</b> Rancher v2.7.0 and later
+    </p>
+    <dd>
+        Helm charts that can extend and enhance the Rancher UI, and tailor Rancher to your specific environment. Some Extensions require manual installation on a cluster, while others are built into Rancher. Built-in Extensions include Fleet and Harvester. Manually installed Extensions inlcude Kubewarden and Elemental.  
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Apps & Marketplace, Catalogs, Integrations</i>
+    </p>
+</dl>
+
+## H
+
+<dl>
+    <dt>
+        Hosted cluster
+    </dt>
+    <dd>
+        A managed cluster that is hosted in the cloud, by a Kubernetes provider. Rancher supports a variety of hosted Kubernetes providers, including Amazon Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), and Google Kubernetes Engine (GKE). Rancher integrates with the hosted Kubernetes provider's cloud APIs, so that you can manage the cluster from the Rancher UI.
+    </dd>
+    <p>
+        <b>Related terms:</b> <i>Managed cluster</i>
+    </p>
+</dl>
+
+## I
+
+<dl>
+    <dt>
+        Imported cluster
+    </dt>
+    <p>
+        <b>Versions:</b> Rancher 2.0–v2.4
```

Comment:
```suggestion
        <b>Versions:</b> Rancher v2.0–2.4
```

The instance under `Apps & Marketplace` (~line 37) uses a `v` only for the first version reference , i.e., `<b>Versions:</b> Rancher v2.5.0–2.6.5`. We should be consistent.